.gitignore

```diff
@@ -2,5 +2,5 @@
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
 
-# Dot ENV file
-.env
\ No newline at end of file
+# Order JSON File
```

Comment:
I believe that the .gitignore could be updated to reflect something like this:
```
# ignore the order files,
*.json
# except exampleorder.json
!exampleorder.json
```
Just so folks don't accidently check-in credit card information and display it public. Try and make it as bulletproof as possible